labrad/server.py

```diff
@@ -197,6 +200,13 @@ def __init__(self):
         self.signals = []
         self.contexts = {}
 
+        self.__protocol = None
+        self.__async_client = None
+        self.__sync_client = threading.local()
```

Comment:
What do the double-underscores represent in these names versus single underscores?